static/app/components/searchSyntax/parser.tsx

```diff
@@ -804,6 +830,10 @@ export type SearchConfig = {
    * Keys considered valid for date filter types
    */
   dateKeys: Set<string>;
+  /**
+   * Disallow wildcards in free text search
```

Comment:
It's more than just free text right, it's disallowing wildcards in free text AND text filters

Reply:
oh yes...good catch here...will update this comment